src/libgriddyn/src/griddyn_generator_management.cpp

```diff
@@ -0,0 +1,46 @@
+#include "griddyn_interface.hpp"
+
+griddyn_generator* griddyn_bus_generator_i(griddyn_sim const* ctx, griddyn_bus const* bus, griddyn_idx_t generator_idx)
```

Comment:
What does the _i  mean?

Reply:
It means I couldn't come up with a better name for array access. Do you have any suggestions?
 
(I don't really like this for generators/loads, since it has to go through the bus pointer map. A hypothetical `griddyn_bus_i_generator_i` wouldn't require that, but would also want better naming.)